### DISCUSSION TITLE
Multi monitor support

### DISCUSSION THREAD

**Discussion by boschet** [dendronhq/dendron#914] [RFCs/Ideas]
I just got the idea/impression that with progress on the graph an tagging features more and more screen space is needed. A nice enhancement would be the addition of some kind of detachable graph view/tag view/etc. from the main window so it can be use on another display. 
Is this even possible?

**Comment by kevins8** [comment]
So vscode supports dragging tabs across multiple windows like in chrome.
Would that meet your use case?

**Reply by boschet** [reply]
In principle this would be fine, but it is not supported as of right now, right? At least it does not work for me. 
I could be difficult to keep track of all the oopen files tough. For example opening a note through the graph view messes with the screen layout as it opens the text where the graph is opened. Having the graph on a different screen one needs to move the tab to the "editing" screen to get side by side preview etc.

**Comment by xypnox** [comment]
Along with the graph, the multi-window/monitor support can also improve
quick note taking as well.

I usually detach the currently opened note from code-workspace window to a
separate window(ctrlK O), use zen mode, and split it with the meeting
browser window.

The problem with detaching the tab is that the dendron features such as
link auto-complete do not work in the detached window.

It would be nice if dendron notes could be edited in different windows with
the same feature-set as available in the code-workspace window.
